tests/tasks/verify_default_keyfile.yml

```diff
@@ -0,0 +1,9 @@
+---
+- name: Verify the default keyfile works
+  shell: >
+    cryptsetup open --test-passphrase "{{ nbde_client_test_device }}"
+    --key-file "{{ nbde_client_test_keyfile}}"
+  ignore_errors: yes
+  register: nbde_keyfile
```

Comment:
please use `nbde_client_keyfile`

Reply:
Fixed this and the other occurrences. Thanks.